Makefile

```diff
@@ -44,7 +45,7 @@ endif
 
 LOCAL_VOLUME_ARGS := -v$(CURDIR):/src:delegated -v $(GOPATH):/go:delegated
 GOPATH_WD_OVERRIDES := -w /src -e GOPATH=/go
-IMAGE_BUILD_FLAGS := -e CGO_ENABLED=0,GOOS=linux,GOARCH=${GOARCH}
```

Comment:
In case you were wondering why this was never caught: Scanner was (and still is) traditionally built in a amd64 linux container, so GOOS and GOARCH were set correctly anyway. Also, I think CGO_ENABLED defaults to 0 anyway